libphal/phal_sbe.C

```diff
@@ -220,17 +222,24 @@ bool isDumpAllowed(struct pdbg_target *proc)
 
 sbeError_t captureFFDC(struct pdbg_target *proc)
 {
-	// get SBE FFDC info
-	bufPtr_t bufPtr;
-	uint32_t ffdcLen = 0;
-	uint32_t status = 0;
-
-	// Log SBE debug data.
-	logSbeDebugData(proc);
+	// only for primary processor SBE run p10_extract_sbe_rc hwp and
+	// perform recovery actions
+	if (isPrimaryProc(proc)) {
+		sbeErrorHandler(proc);
```

Comment:
How we will collect SBE FFDC if any exception from sbeErrorHandler?

Reply:
my idea for now is to catch the exception in phal_error.cpp in openpower-proc-control. phal_error.cpp has the code now to collect sbe_ffdc

Reply:
as discussed for now we will throw SBE_CMD_TIMEOUT error for any exception in sbeErrorHandler, till we had catered for all the error cases in proc-control and phosphor-logging